site/docs/v1/tech/apis/reactor.adoc

```diff
@@ -0,0 +1,124 @@
+---
+layout: doc
+title: Reactor APIs
+description: APIs for activating, deactivating, and retrieving the status of a license.
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+This page contains the APIs for Reactor, FusionAuth's license system. Reactor is used to activate features based upon your licensing tier. Here are the APIs:
+
+* <<Activate Reactor license>>
+* <<Retrieve Reactor status>>
+* <<Regenerate Reactor license>>
+* <<Deactivate Reactor license>>
+
+
+== Activate Reactor license
+
+This API is used to activate a Reactor license.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Activate a Reactor license
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/reactor#
+--
+
+==== Request Body
+
+[.api]
+[field]#licenseId# [type]#[String]# [required]#Required#::
+The license Id to to activate.
+
+[field]#license# [type]#[String]# [optional]#Optional#::
+The Base64 encoded license value. This value is necessary in an link:https://en.wikipedia.org/wiki/Air_gap_(networking)[air gapped] configuration where outbound network access it not available.
+
+=== Response
+
+This API does not return a JSON response body.
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
```

Comment:
This standard response code has the line: `The request was successful. The response will contain a JSON body.`

You might need to modify it to omit the last sentence. Suggest you add an ifdef and a parameter: `no_success_json_body` or similar.

Reply:
Here's the delete verbiage: `The request was successful. The response will be empty.`